### ISSUE TITLE
En anglais?

### ISSUE THREAD

**Issue by Vict0rSch** [climanifeste/climanifeste#106]
Quelle est votre opinion sur l'internationalisation de cette démarche?

**Comment by jffourmond** [comment]
Le texte actuel est écrit pour un public français. Chiffres, articles de presse, entreprises citées : tout y est spécifique à la France. Je pense qu'une adaptation à un autre pays serait plus pertinente qu'une simple traduction dans une autre langue. Mais si tu es motivé pour faire une traduction en anglais @Vict0rSch, on peut envisager de la mettre sous la version française. Quelle était ton idée ?

En parlant de ça, @supertanuki connait quelqu'un qui serait intéressé pour faire une version en portugais pour le Brésil. Des news de ce côté ?

**Comment by supertanuki** [comment]
Oui une version en portugais pour le Brésil va peut être naître et sans doute portée par les brésiliens eux mêmes.
Je suis d’accord avec @jffourmond pour que chaque pays intéressé ne traduise pas mais adapte le manifeste à ses spécificités.

**Comment by Vict0rSch** [comment]
Je suis d'accord sur le fait qu'il ne faille pas simplement traduire mais adapter; en attendant je trouve que c'est plus "engaging" d'avoir une version traduite où les sources seraient adaptées au fur et à mesure parce qu'en l'état ça ne suppose pas simplement l'intérêt de gens d'autres pays mais aussi leur capacité à comprendre le message.

Donc une version hybride serait de faire la traduction en fonction des compétences des volontaires présents en précisant que c'est un work in progress qui nécessite non seulement une adaptation du texte mais aussi des sources en fonction des spécificités de chaque communauté

**Comment by Pascal-KOTTE** [comment]
Je pense que poser une version traduite en anglais, serait relativement universel pour le monde ICT, et une bonne idée. Mais qui peut faire ?
Dès que je suis à la retraite, je m'y colle... SOS volontaires, les hacktivistes croulent, il faut de l'aide.
Le truc c'est de trouver les version EN des textes de références ou autres initiatives plus universelles, ou juste existantes en EN, tu as raison @jffourmond -- Hum, approcher les ONG et des gens comme Tim Berner-Lees... ?
